apps/dine-pleiepenger/src/i18n/nb.ts

```diff
@@ -27,6 +27,10 @@ export const nb = {
     'svarfrist.tittel': 'Saksbehandlingstid',
     'svarfrist.forventetBehandlingstid':
         'Forventet behandlingstid er {saksbehandlingstidUker} uker fra vi fikk søknaden din.',
+    'svarfrist.fristPassert.1':
+        'Vi jobber fortsatt med å behandle søknaden din. Noen ganger kan det ta lengre tid, hvis saken er spesielt krevende eller hvis vi har ventet på dokumentasjon.',
```

Comment:
Denne setningen var litt klønete bygget opp synes jeg, spesielt med kommaet før "hvis". Har vi påvirkning på disse tekstene selv?

Forslag fra en som ikke er noen tekstforfatter:

```suggestion
        'Vi jobber fortsatt med å behandle søknaden din. Du trenger ikke foreta deg noe. Noen ganger kan behandlingen ta lengre tid enn antatt, spesielt hvis saken er spesielt krevende eller vi har ventet på dokumentasjon.',
```
Merk jeg slenger med en liten delsetning om hva bruker trenger (eller ikke trenger) å gjøre.

Reply:
Ja og nei - vi har gjennomganger og så blir det bestemt der. Tror jeg lar denne være som den er.

Reply:
Okei!